netbox/extras/graphql/types.py

```diff
@@ -186,6 +187,15 @@ class SubscriptionType(ObjectType):
     user: Annotated["UserType", strawberry.lazy('users.graphql.types')] | None
 
 
+@strawberry_django.type(
+    models.TableConfig,
+    filters=TableConfigFilter,
+    pagination=True
+)
+class TableConfigType(ObjectType):
+    user: Annotated["UserType", strawberry.lazy('users.graphql.types')] | None
```

Comment:
I'm only seeing `changelog`, `display`, `class_type`, and `user` as available output types for this type. Is that intentional?

Reply:
Good catch. The type' decorator is missing `fields='__all__'`.